subjects/gaming/stealth-boom/audit/README.md

```diff
@@ -2,68 +2,90 @@
 
 #### Functional
 
-###### Does the map contain places for the player to hide from enemies?
+##### Main Menu
```

Comment:
The title sections like :

```md
##### Main Menu
##### Enemies
##### Map / Level
...
```

they use 5 hashes, and this normally we only use to give students commands like this example:

```md
##### Check the Repo content

A `README.md` file and all files used to create, delete and manage the student infrastructure must be submitted in the repo.

###### Are all the required files present?
```

You can see we use 5 hashes for the command and 6 hashes for the actual yes/no question.
So in my opinion we should make the functional go away  and use 4 hashes for those topics or we can use ** 

```md
**Main Menu**
**Enemies**
**Map / Level**
...
```

@Oumaimafisaoui what do you think?

Reply:
i removed functional and made all topics only 4 hashes